src/tests/ftest/osa/offline_extend.py

```diff
@@ -6,8 +6,7 @@
 """
 from osa_utils import OSAUtils
 from daos_utils import DaosCommand
-from test_utils_pool import TestPool
-from test_utils_base import LabelGenerator
+from test_utils_pool import add_pool
```

Comment:
(pylint-syntax-error) Cannot import 'test_utils_pool' due to syntax error 'invalid syntax (<unknown>, line 53)'